velox/exec/tests/TableWriteTest.cpp

```diff
@@ -2262,6 +2264,114 @@ TEST_P(AllTableWriterTest, tableWriteOutputCheck) {
   ASSERT_EQ(obj[TableWriteTraits::kLifeSpanContextKey], "TaskWide");
 }
 
+TEST_P(AllTableWriterTest, columnStatsDataTypes) {
+  std::vector<RowVectorPtr> input;
+  input.reserve(6);
+  input.push_back(makeRowVector(
+      rowType_->names(),
```

Comment:
Does this cover all the supported column data and stats type combinations? thanks!